frontend/src/stories/ApplicationSummaryTable.stories.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import ApplicationSummaryTable from '../components/ApplicationSummaryTable';
+
+export default {
+  title: 'ITVR/ApplicationFormDetails',
+  component: ApplicationSummaryTable
+};
+
+const Template = (args) => <ApplicationSummaryTable {...args} />;
+export const Default = Template.bind({});
```

Comment:
Does this still have data returned now using the query client? It prob makes sense to use create a component for the data table taking the api data as input and then using that with storybook.